### ISSUE TITLE
Issue with JAWS screen reader

### ISSUE THREAD

**Issue by Kenji776** [pozil/sfdc-ui-lookup-lwc#121]
Hello! First off, just want to say thank you for this awesome component, it really is great and much appreciated. I am however, having one issue that I don't know if you might have any insight on. We are doing some accessibility testing/review using the JAWS screen reader software that is used by many visually impaired folks. We have found that when JAWS is open, for some reason we are unable to interact with the drop down results via keyboard. It is as if the elements simply do not exist. If you use the arrow keys to move up and down it simply interacts with the other elements on the page instead of the results in the list. I've looked around and didn't see any aria tags that would be the likely culprit (like aria-hidden, or a negative tab index) so I'm not sure why it's behaving differently. I'll keep playing around and see if I can find a fix and if so I'll report it here but if you have any ideas it would be much appreciated cause I've been trying to fix this for a while and am running out of ideas. Thanks!

**Comment by pozil** [comment]
Hi @Kenji776 and thanks for the feedback. There's no doubt that a11y is super important.
I was working with the assumption that the LDS blueprint had everything that I needed a11y-wise but it turns out that the lookup needed a few fixes. I made those and enforced a11y testing with [sa11y](https://github.com/salesforce/sa11y) as part of CI.
Please try the latest release (v2.18.0) and let me know if this fixes the issue.

**Comment by Kenji776** [comment]
@pozil Thank you so much for the reply. I'll give this new version a shot and see how it goes.

**Comment by Kenji776** [comment]
The updated version is working perfectly, much appreciated. Great work!

**Comment by pozil** [comment]
Thanks for confirming. I'm glad that this gave me the opportunity to improve the component.
